src/main/java/roomescape/api/ReservationController.java

```diff
@@ -0,0 +1,57 @@
+package roomescape.api;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import roomescape.api.dto.ReservationRequestDto;
+import roomescape.api.dto.ReservationResponseDto;
+import roomescape.repository.ReservationRepositoryImpl;
+import roomescape.service.ReservationService;
+
+import java.util.List;
+
+@Controller
+public class ReservationController {
+
+    private final ReservationService reservationService;
+
+    public ReservationController() {
+        this.reservationService = new ReservationService(new ReservationRepositoryImpl());
+    }
+
+    @GetMapping("/reservation")
+    public String reservationPage() {
+        return "reservation";
+    }
+
+    @GetMapping("/reservations")
+    @ResponseBody
+    public ResponseEntity<List<ReservationResponseDto>> readReservations() {
+        return ResponseEntity
+                .status(HttpStatus.OK)
+                .body(
+                        reservationService.readReservations().stream().
+                                map(ReservationResponseDto::fromEntity)
+                                .toList()
```

Comment:
stream과 정적 팩터리 메서드 활용으로 로직이 엄청 깔끔하네요 👍

```suggestion
                      reservationService.readReservations().stream()
                              .map(ReservationResponseDto::fromEntity)
                              .toList()
```